app/org/sagebionetworks/bridge/models/VersionHolder.java

```diff
@@ -1,10 +1,12 @@
 package org.sagebionetworks.bridge.models;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class VersionHolder {
 
     private final Long version;
     
-    public VersionHolder(Long version) {
+    public VersionHolder(@JsonProperty("version")Long version) {
```

Comment:
This is usually a readonly property (we create and return it but don't deserialize it), so if you aren't deserializing it anywhere, you shouldn't need this annotation.